src/components/link/ExternalLink.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react"
```

Comment:
This compoinent is not needed -> we can remove it and use just **<A>** which is already created. Just remember to add **outside** property for it.